x-pack/plugins/secops/public/components/draggables/index.tsx

```diff
@@ -17,20 +17,77 @@ import { Provider } from '../timeline/data_providers/provider';
 export interface DefaultDraggableType {
   id: string;
   field: string;
-  value: string | null | undefined;
+  value?: string | null;
   name?: string | null;
   queryValue?: string | null;
   children?: React.ReactNode;
+  tooltipContent?: React.ReactNode;
 }
 
+/**
+ * Only returns true if the specified tooltipContent is exactly `null`.
+ * Example input / output:
+ * `bob -> false`
+ * `undefined -> false`
+ * `<span>thing</span> -> false`
+ * `null -> true`
+ */
+export const tooltipContentIsExplicitlyNull = (tooltipContent?: React.ReactNode): boolean =>
+  tooltipContent === null; // an explicit / exact null check
+
+/**
+ * Derives the tooltip content from the field name if no tooltip was specified
+ */
+export const getDefaultWhenTooltipIsUnspecified = ({
+  field,
+  tooltipContent,
+}: {
+  field: string;
+  tooltipContent?: React.ReactNode;
+}): React.ReactNode => (tooltipContent != null ? tooltipContent : field);
+
+/**
+ * Renders the content of the draggable, wrapped in a tooltip
+ */
+const Content = pure<{
+  children?: React.ReactNode;
+  field: string;
+  tooltipContent?: React.ReactNode;
+  value?: string | null;
+}>(({ children, field, tooltipContent, value }) =>
+  !tooltipContentIsExplicitlyNull(tooltipContent) ? (
```

Comment:
I can read the double negative, but I would of just had a single call with the native expression like so

```ts
// explicit null check 
tooltipContent !== null ? 
```

since this is the only place this function is being used and not done the extra tests. But I appreciate the extra semantic thoroughness of what you have.